# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/d3d11/MLGDeviceD3D11.cpp
# Commit: 1d9384906287
# Full Hash: 1d93849062870a8d492eeef2150ecea174cb6223
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-26 21:12:53
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r f5ebdcca640d -r 1d9384906287 gfx/layers/d3d11/MLGDeviceD3D11.cpp
--- a/gfx/layers/d3d11/MLGDeviceD3D11.cpp	Sun May 26 14:29:56 2019 +0000
+++ b/gfx/layers/d3d11/MLGDeviceD3D11.cpp	Sun May 26 14:30:14 2019 +0000
@@ -321,7 +321,7 @@
 
   if (!mRT) {
     MLGRenderTargetFlags flags = MLGRenderTargetFlags::Default;
-    if (gfxPrefs::AdvancedLayersEnableDepthBuffer()) {
+    if (StaticPrefs::AdvancedLayersEnableDepthBuffer()) {
       flags |= MLGRenderTargetFlags::ZBuffer;
     }
 